metrics.proto

```diff
@@ -121,3 +122,15 @@ message RdmaEntry {
 	uint32 hca_handles = 2;
 	uint32 hca_objects = 3;
 }
+
+message NetworkStat {
+	string name = 1;
+	uint64 RxBytes = 2;
```

Comment:
these should be lowercase:

`rx_bytes` `tx_bytes`

Reply:
done.